install_config/persistent_storage/persistent_storage_iscsi.adoc

```diff
@@ -84,3 +83,37 @@ namespace. Any attempt to access a persistent volume across a namespace causes
 the pod to fail.
 
 Each iSCSI LUN must be accessible by all nodes in the cluster.
+
+[[iscsi-multipath]]
+
+=== iSCSI Multipathing
+For iSCSI based storage, you can configure multiple paths by using same IQN for more than one IP addresses. Multipathing ensures access to the persistent volume when one or more of the components in a path fail.
+
+To specify multi-paths in pod specification use the `portal` field. For example:
+
+====
+[source, yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: iscsipd
+spec:
+  containers:
+  - name: iscsipd-rw
+    image: kubernetes/pause
+    volumeMounts:
+    - mountPath: "/mnt/iscsipd"
+      name: iscsipd-rw
+  volumes:
+  - name: iscsipd-rw
+    iscsi:
+      targetPortal: 10.0.2.15:3260
+      portals: ['10.0.2.16:3260', '10.0.2.17:3260', '10.0.2.18:3260'] <1>
+      iqn: iqn.2016-04.test.com:storage.target00
+      lun: 0
+      fsType: ext4
+      readOnly: true
```

Comment:
I think mentioning 'persistent Volume spec' is preferred  than 'spec' directly in pod spec. @rootfs any thoughts.